okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -597,11 +597,13 @@ public String fragment() {
     return fragment;
   }
 
-  /** Returns the URL that would be retrieved by following {@code link} from this URL. */
+  /**
+   * Returns the URL that would be retrieved by following {@code link} from this URL. Equivalent
```

Comment:
It was missing before, but maybe mention that this returns `null` if the link cannot be resolved?